src/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseException.cs

```diff
@@ -68,20 +67,11 @@ protected LicenseException(SerializationInfo info, StreamingContext context) : b
         public Type LicensedType { get; }
 
         /// <summary>
-        ///     Need this since Exception implements ISerializable and we have fields to save out.
+        ///     Need this since Exception implements ISerializable.
         /// </summary>
-        [SecurityPermission(SecurityAction.Demand, SerializationFormatter = true)]
         public override void GetObjectData(SerializationInfo info, StreamingContext context)
         {
-            if (info == null)
-            {
-                throw new ArgumentNullException(nameof(info));
-            }
-
-            info.AddValue("type", LicensedType);
-            info.AddValue("instance", _instance);
-
-            base.GetObjectData(info, context);
+            throw new PlatformNotSupportedException();
```

Comment:
There seems to be some inconsistency here.  If the type was already overriding GetObjectData, then calling it will throw a PNSE, but if it wasn't already overriding GetObjectData, it won't throw?